tests/auto/util/tst_util.cpp

```
@@ -1446,5 +1480,101 @@ void tst_util::findBinaryInPathStringLiteral() {
+           "updateEnv should set PASSWORD_STORE_CHARACTER_SET");
+}
+
+void tst_util::updateEnvEmptyCustomCharsetFallsBackToAllChars() {
+  TestPass pass;
+  PasswordConfiguration config = QtPassSettings::getPasswordConfiguration();
+  config.selected = PasswordConfiguration::CUSTOM;
+  config.Characters[PasswordConfiguration::CUSTOM] = QString();
+  QtPassSettings::setPasswordConfiguration(config);
+  struct ConfigRollback {
+    PasswordConfiguration value;
+    ~ConfigRollback() { QtPassSettings::setPasswordConfiguration(value); }
```

Clang (14.0.6)

[error] use '= default' to define a trivial destructor

(modernize-use-equals-default,-warnings-as-errors)